roles/v2/pool/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
```

Comment:
Remove also this one.

Reply:
Done, you are right! I think the rebase from main to dev put this there because I made some changes to it.